frontend/src/metabase/setup/constants.ts

```diff
@@ -3,8 +3,9 @@ export const LOCALE_TIMEOUT = 300;
 export const WELCOME_STEP = 0;
 export const LANGUAGE_STEP = 1;
 export const USER_STEP = 2;
-export const DATABASE_STEP = 3;
-export const PREFERENCES_STEP = 5; // backward compatibility for analytics
+export const USAGE_STEP = 3;
+export const DATABASE_STEP = 4;
+export const PREFERENCES_STEP = 5;
```

Comment:
This is a breaking change in the analytics for some questions/queries, I have an ongoing discussion on slack and I will address it with a later task

Reply:
Yes sorry: https://metaboat.slack.com/archives/C01MC2CKW2G/p1706719539434809